lucent/optvis/render.py

```diff
@@ -130,7 +202,7 @@ def closure():
         show(tensor_to_img_array(image_f()))
     elif show_image:
         view(image_f())
-    return images
+    return images #, smooth_pixelwise_grads
```

Comment:
Consider adding an argument that allows to return the smoothed gradients as well?